<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AndroidSizeWidget</class>
 <widget class="QWidget" name="AndroidSizeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="1">
    <widget class="QLineEdit" name="xhdpiValue">
     <property name="placeholderText">
      <string>2.00x</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="xxxhdpiLabel">
     <property name="text">
      <string>XXXHDPI :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="xxhdpiLabel">
     <property name="text">
      <string>XXHDPI :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="xxhdpiValue">
     <property name="placeholderText">
      <string>3.00x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="ldpiLabel">
     <property name="text">
      <string>LDPI :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mdpiValue">
     <property name="placeholderText">
      <string>1.00x</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="hdpiLabel">
     <property name="text">
      <string>HDPI :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="hdpiValue">
     <property name="placeholderText">
      <string>1.50x</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="xhdpiLabel">
     <property name="text">
      <string>XHDPI :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ldpiValue">
     <property name="placeholderText">
      <string>0.75x</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mdpiLabel">
     <property name="text">
      <string>MDPI :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="xxxhdpiValue">
     <property name="placeholderText">
      <string>4.00x</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="ldpiCopyButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources.qrc">
       <normaloff>:/ic/ic_copy</normaloff>:/ic/ic_copy</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="mdpiCopyButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources.qrc">
       <normaloff>:/ic/ic_copy</normaloff>:/ic/ic_copy</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="hdpiCopyButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources.qrc">
       <normaloff>:/ic/ic_copy</normaloff>:/ic/ic_copy</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="xhdpiCopyButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources.qrc">
       <normaloff>:/ic/ic_copy</normaloff>:/ic/ic_copy</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="xxhdpiCopyButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources.qrc">
       <normaloff>:/ic/ic_copy</normaloff>:/ic/ic_copy</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="xxxhdpiCopyButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources.qrc">
       <normaloff>:/ic/ic_copy</normaloff>:/ic/ic_copy</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ldpiValue</tabstop>
  <tabstop>mdpiValue</tabstop>
  <tabstop>hdpiValue</tabstop>
  <tabstop>xhdpiValue</tabstop>
  <tabstop>xxhdpiValue</tabstop>
  <tabstop>xxxhdpiValue</tabstop>
  <tabstop>ldpiCopyButton</tabstop>
  <tabstop>mdpiCopyButton</tabstop>
  <tabstop>hdpiCopyButton</tabstop>
  <tabstop>xhdpiCopyButton</tabstop>
  <tabstop>xxhdpiCopyButton</tabstop>
  <tabstop>xxxhdpiCopyButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
